linkis-engineconn-plugins/engineconn-plugins/hive/src/main/scala/org/apache/linkis/engineplugin/hive/creation/HiveEngineConnFactory.scala

```diff
@@ -69,6 +69,11 @@ class HiveEngineConnFactory extends ComputationSingleExecutorEngineConnFactory w
       hiveConf.setVar(HiveConf.ConfVars.HIVEFETCHOUTPUTSERDE, HiveEngineConfiguration.BASE64_SERDE_CLASS)
       hiveConf.set("enable_fetch_base64","true")
     }
+    //add hive.aux.jars.path to hive conf
+    if(StringUtils.isNotEmpty(HiveEngineConfiguration.HIVE_AUX_JARS_PATH)) {
```

Comment:
Is it better to use isNotBlank here?

Reply:
Yes, it's better to use isNotBlank . I will update it.